app/queries/application_query.rb

```diff
@@ -34,7 +34,7 @@ def notify(kind, title, body)
   end
 
   def notify_errors
-    notify(:error, 'Something went wrong!', error_messages.join(", "))
+    notify(:error, I18n.t("errors.notify_errors"), error_messages.join(", "))
```

Comment:
Can't we use a more specific key? 
```suggestion
    notify(:error, I18n.t("errors.something_went_wrong"), error_messages.join(", "))
```

Reply:
done